src/models/mistral.py

```diff
@@ -0,0 +1,36 @@
+from models.base import BaseChatbot, CompletionResponse
+import requests
+
+class Mistral:
+    
+    def __init__(
+        self, 
+        prompt: str = "", 
+        api_key: str = "", 
+        temp: float = 0,
+    ) -> None:
+        self.prompt = prompt
+        self.api_key = api_key
+        self.temp = temp
+
+    def get_completion(self) -> CompletionResponse:
+
+        try:
+            API_URL = "https://api-inference.huggingface.co/models/mistralai/Mistral-7B-v0.1"
```

Comment:
As discussed in the meeting today, you should implement the chatbot version of mistral so that we can get responses rather than completions

Reply:
As mentioned, I switched the API to Mistral-Instruct